lametro/templates/lametro/index.html

```diff
@@ -141,17 +141,28 @@ <h2>
           serviceUrl: '{% url "autocomplete" %}',
           paramName: 'query',
           transformResult: function(response) {
-            var response = JSON.parse(response);
-            return {
-              suggestions: $.map(response.termHints, function(d) {
-                return {'value': d.name, 'data': d.id};
-              })
+            var res = JSON.parse(response);
+            if (res.status_code == 500 || res.termHints.length < 1) {
+              return {
+                suggestions: [{'value': 'No topics found', 'data': 'na'}]
+              };
+            } else {
+              return {
+                suggestions: $.map(res.termHints, function(d) {
+                    return {'value': d.name, 'data': d.id};
+                  }
+                )
+              };
             }
           },
+          onSearchError: function(query, jqXHR, textStatus, errorThrown) {
+            return textStatus + ' : ' + errorThrown;
```

Comment:
what happens to this message? (does it get console logged or put somewhere we can see it if we try to reproduce the problem?)

Reply:
it's not required and we're not using it—i went ahead and removed it since the errors weren't being utilized.